src/validation/loader.rb

```diff
@@ -31,18 +31,13 @@
 module Metalware
   module Validation
     class Loader < LoadSaveBase
-      def initialize(metalware_config, cache_configure: false)
+      def initialize(metalware_config)
         @config = metalware_config
         @path = FilePath.new(config)
-        @cache_configure = cache_configure
       end
 
       def configure_data
-        if cache_configure
-          @configure_data ||= configure_data_tree
-        else
-          configure_data_tree
-        end
+        @configure_data ||= configure_data_tree
```

Comment:
Ignore my previous comment about caching, just saw this commit. This seems a reasonable change. My only comment is we now have the `Loader` class implicitly caching somethings but not others.

I'm not sure if this a problem or not, but it might lead to some inconsistencies in the future.

Reply:
Thanks for mentioning this, it's worth considering. I think it's probably OK to sometimes cache things and sometimes not - it's pretty standard Ruby to use `||=` to cache some things which won't change and might be slightly slow, which loading the configure data could be as it can now involve loading and parsing/validating several files. From the tests still passing we can see that always caching doesn't change the behaviour (at least the behaviour we care about enough to test) - if we later decide we need to cache more things we can see if that breaks anything and handle it appropriately then.